package.json

```diff
@@ -124,6 +125,7 @@
     "@storybook/manager-webpack5": "~6.4.12",
     "@storybook/react": "~6.4.12",
     "@svgr/webpack": "^5.4.0",
+    "@testing-library/cypress": "^8.0.3",
```

Comment:
Is this being used anywhere

Reply:
Not anymore. I just removed it. Thanks for the reminder :)